### ISSUE TITLE
very long time skips with FF & RW with left/right arrows

### ISSUE THREAD

**Issue by SECURIOlab** [yuliskov/SmartYouTubeTV#58]
bug or feature ? watch the video: https://photos.app.goo.gl/5OXu0ODUQJ1iGjhx1
just 7 clicks to seek all the song? It is not possible to jump another time position because it is very long period skiping :(  Much better if FF and RW funcion wil be exactly same as original YT TV app.

**Comment by yuliskov** [comment]
@SECURIOlab agree with you. This need to be fixed in the next release.

**Comment by yuliskov** [comment]
Done: https://github.com/yuliskov/SmartYouTubeTV/releases/tag/6.12.43

**Comment by SECURIOlab** [comment]
you are the fastest programmer ever seen ;]
Now just button of one click to rewind to the beginning of video

**Comment by yuliskov** [comment]
@SECURIOlab thanks, is it mean that problem has been fixed?

**Comment by SECURIOlab** [comment]
yes, fixed great. Long film video must now longer rewind by holding the arrows but still can use USB mouse. Or you need to implement accelerated rewind of long time videos.

**Comment by yuliskov** [comment]
@SECURIOlab I've no idea how to apply seek acceleration. However, current setting are:
5 sec step: 0 - 10 min videos
10 sec step: 10 - 60 min videos
30 sec step: 60 - ... min videos
